be/src/exprs/runtime_filter.h

```diff
@@ -430,14 +508,68 @@ class RuntimeBloomFilter final : public JoinRuntimeFilter {
     bool right_close_interval() const { return _right_close_interval; }
 
     void evaluate(Column* input_column, RunningContext* ctx) const override {
-        if (_num_hash_partitions != 0) {
-            return _t_evaluate<true>(input_column, ctx);
+        if (!_hash_partition_bf.empty()) {
+            _t_evaluate<true>(input_column, ctx);
+        } else {
+            _t_evaluate<false>(input_column, ctx);
+        }
+    }
+
+    // `hash_parittion` parameters means if this runtime filter has multiple `simd-block-filter` underneath.
+    // for local runtime filter, it only has once `simd-block-filter`, and `hash_partition` is false.
+    // and for global runtime filter, since it concates multiple runtime filters from partitions
+    // so it has multiple `simd-block-filter` and `hash_partition` is true.
+    // For more information, you can refers to doc `shuffle-aware runtime filter`.
+
+    template <bool multi_partition>
```

Comment:
parameter's name is not consistent with it in comments, seems we should change `hash_partition` to `multi_partition` in comments?